[RUM-9722] Add remote configuration instructions to all auto injection setup (#29005)
DataDog/documentation: content/en/real_user_monitoring/browser/setup/server/apache.md

## Set up your RUM application

To automatically instrument your RUM application:

> 1. In Datadog, navigate to the [**Digital Experience > Add an Application Page**][4] and select the JavaScript (JS) application type.

Reviewer: Not really related to this PR, but I just noticed and we might want to make it more clear: 

<img width="568" alt="image" src="https://github.com/user-attachments/assets/2e7e9b51-d64a-4563-8cc7-5bb998763784" />

<img width="153" alt="image" src="https://github.com/user-attachments/assets/c086706b-c4be-4aa4-84d2-7c0e9883b903" />

`Digital Experience > Add an Application` doesn't exist. It is obvious enough that I don't think anyone will be confused, but we might want to be more explicit.

This appears in all cards BTW

Author: Yes, right. I fixed it.

2. Select **Auto-Instrumentation** and **Apache httpd**.
3. Configure your application parameters. See [guidance on configuring sampling][5].
   {{< img src="real_user_monitoring/browser/auto-instrumentation/auto-instrumentation-configuration.png" alt="Configure your application" style="width:90%;">}}

4. Copy and run the installer command to load the Datadog httpd Module with the RUM SDK Injector onto httpd.
   {{< img src="real_user_monitoring/browser/auto-instrumentation/httpd-code-snippet.png" alt="Auto-instrumentation Configuration" style="width:90%;">}}

5. After the installer successfully installs the SDK Injector, restart Apache HTTP Server to begin collecting RUM sessions.
6. (Optional) To verify the module is successfully injecting the RUM Browser SDK into HTML pages, check the error logs for relevant messages. The module logs important steps during the injection process. Ensure that Apache HTTP Server is configured with at least the `info` log level.

> Alternatively, you can [manually](#alternative-manual-installation) install and configure the module.

Reviewer 2: ```suggestion
Alternatively, you can [manually](#alternative-installation-method) install and configure the module.
```

> Alternatively, you can also [manually](#alternative-manual-installation) install and configure the module.

Reviewer 2: ```suggestion
Alternatively, you can [manually](#optional-manual-installation) install and configure the module.
```

3. Copy and paste the configuration snippet to your `/opt/datadog-httpd/datadog.conf` file.

## Troubleshooting

### RUM is not injected

If you notice that RUM is not being injected into HTML pages, consider the following potential causes:

- **Content-Type mismatch**: RUM is injected only into HTML pages. If the `Content-Type` header does not correctly indicate `text/html`, the injection is skipped.

### Limitations

See other [Limitations][1].

## Uninstall

To manually remove RUM from your auto-instrumented web server:

1. Locate the Apache (`httpd`) configuration file by running `httpd -V`. Depending on the Linux distribution used, this binary file could be named `http`, `apachectl`, `apache2` or `apache2ctl`. The following steps use `httpd` as an example. In this instance, the file location could be: `/usr/local/apache2/conf/httpd.conf`.
2. At the end of the httpd configuration file, remove the line: `Include /opt/datadog-httpd/datadog.conf`.
3. Delete the directory `/opt/datadog-httpd/` and all of its contents.
4. Restart or reload Apache httpd.

> ## (Alternative) Manual installation

Reviewer 2: ```suggestion
## Alternative installation method
```

> ## (Alternative) Manual installation

Author: :) I first wrote "Optional" but after reconsidering, I chose “Alternative” because to me it better conveys the idea: Manual installation is not compatible with automatic setup, so it’s not optional. It’s an alternative method. Wdyt?

> ## (Alternative) Manual installation

Reviewer 2: hmm i agree with that assessment. let's go with alternative installation method. i believe this is a common way of handling it.
